test/e2e/app-dir/dynamic-io-errors/fixtures/sync-attribution/guarded-async-guarded-clientsync/app/page.tsx

```diff
@@ -0,0 +1,38 @@
+import { cookies } from 'next/headers'
+import { Suspense } from 'react'
+
+import { SyncIO } from './client'
+
+export default async function Page() {
+  return (
+    <main>
+      <section>
+        In This test we have a Client Component that accesses sync IO in a
+        Suspense boundary and a Server Component that accesses Request data
+        without a Suspense Boundary. We expect this to error because of the
+        Request data access and need to assert that the error does not
+        incorrectly indicate that sync IO is the problem.
```

Comment:
The server component does have a Suspense boundary, and we do not expect an error.